src/main/kotlin/wtf/votebot/bot/config/Config.kt

```diff
@@ -37,6 +39,6 @@ interface Config {
     }
 
     fun environmentType(): Environment {
-        return Environment.valueOf(environment.toLowerCase())
```

Comment:
Why not using single line methods here: https://subscription.packtpub.com/book/application_development/9781787126367/4/ch04lvl1sec49/single-expression-functions

Reply:
Done